packages/core/src/runtime/resolveStyleRules.test.ts

```diff
@@ -11,6 +12,19 @@ function getFirstClassName([resolvedClassesForSlot]: [CSSClassesMap, CSSRulesByB
 }
 
 describe('resolveStyleRules', () => {
+  describe('unsupported css properties', () => {
+    it.each(Object.keys(UNSUPPORTED_CSS_PROPERTIES))(
+      'strips unsupported `%s` css property when not in production',
+      property => {
+        // Doesn't matter what the value is, just that the resulting objects are empty
+        const res = resolveStyleRules({ [property]: 'dummy' as unknown as undefined });
+        expect(res).toHaveLength(2);
+        expect(res[0]).toEqual({});
+        expect(res[1]).toEqual({});
+      },
+    );
+  });
```

Comment:
Can you please add a fixture to Babel plugin too?

Reply:
Done, I added the code in JS because our types are working.... alternatively I could have used ts with `@ts-nocheck`

Reply:
Can we use TS and `ts-expect-error` instead?

Reply:
I had to use it for every single property ☹️